src/components/aboutUs.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 const AboutUs = () => (
-  <div className="m-2 bg-red-600 font-lato w-full p-4">
+  <div className="m-2 bg-red-600 font-lato  p-4">
```

Comment:
Remove extra space between classes, running linting before opening pull request will catch this!